lib/gamera/modules/reloadable.rb

```diff
@@ -0,0 +1,24 @@
+module Gamera
+  module Reloadable
+    # Includes a method for reloading the page to wait for a condition to pass.
+
+    include Capybara::DSL
+
+    # A method to wait for slow loading data on a page. Useful, for example,
+    # when waiting on a page that shows the count of records loaded via a slow
+    # web or import.
+    #
+    # @param retries [Integer] Number of times to reload the page before giving up
+    # @param allowed_errors [Array] Array of errors that trigger a refresh, e.g.,  if an ExpectationNotMetError was raised during an acceptance test
+    # @param block [Block] The block to execute after each refresh
+    def with_refreshes(retries, allowed_errors = [RSpec::Expectations::ExpectationNotMetError], &block)
+      retries_left ||= retries
+      visit
```

Comment:
`visit` is an implicit dependency on `Visitable`.  I'd like to make that explicit.

We could explicitly `include Gamera::Visitable`.  Or implement a `visit` method which raises an exception saying you need to implement a `visit` of your own or include ours.

We could also put `with_refreshes` in `Visitable`.  That's saying that we know visitable pages are reloadable.

I think I like the last option better, but I have to say I also like the separation of concerns expressed here in making it a separate module.